.github/workflows/chart.yaml

```diff
@@ -6,11 +6,39 @@ on:
       - 'main'
     paths:
       - 'charts/**'
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'charts/**'
 
 jobs:
+  helm-verify:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Find new version
+        id: new_version
+        run: |
+          NEW_VERSION=$(yq e '.version' charts/gitops-server/Chart.yaml)
+          echo "::set-output name=version::$NEW_VERSION"
+      - name: Find old version
+        id: old_version
+        run: |
+          git checkout ${{ github.event.pull_request.head.sha }}
+          OLD_VERSION=$(yq e '.version' charts/gitops-server/Chart.yaml)
+          echo "::set-output name=version::$OLD_VERSION"
+      - name: Alert about the need to change chart version
+        run: echo "This PR changed the helm chart. You need to change chart version when you change the chart"
```

Comment:
I think this needs to return a non-zero status otherwise no one's going to see this check fire (I think)

Reply:
:facepalm: